### ISSUE TITLE
Allow options on install-modules option

### ISSUE THREAD

**Issue by ylavoie** [shogo82148/actions-setup-perl#701]
Related to #684 & #686

Install options should allow user to add options for the installers, to grant possibility to cache features or develoment mode.

Install modules does a great job to install modules listed in the main part of `cpanfile`.
The user still need to implement caching to install features.

For example:
```
        uses: shogo82148/actions-setup-perl@v1.11.0
        with:
          perl-version: 5.32
          install-modules-with: cpm
            - with-develop
            - feature=f1,f2
```
would allow a single invocation, let the user benefit from the implemented caching and install development and features f1 & f2.

**Comment by shogo82148** [comment]
I'm working on this, and it is available on [v1.12.0-rc1](https://github.com/shogo82148/actions-setup-perl/releases/tag/v1.12.0-rc1).

```yaml
steps:
- uses: actions/checkout@v2
- uses: shogo82148/actions-setup-perl@v1.12.0-rc1
  with:
    perl-version: '5.32'
    install-modules-with: cpm
    install-modules-args: --with-develop --with-configure
- run: prove -lv t
```

please try it, and give me a feedback.

**Comment by ylavoie** [comment]
```
      - name: Setup Perl environment
        uses: shogo82148/actions-setup-perl@v1.12.0-rc1
        with:
          perl-version: 5.32
          install-modules-with: cpm
          install-modules-args: --with-develop --feature=starman --feature=latex-pdf-ps --feature=openoffice --feature=xls --feature=edi
```
worked wonderfully, installed the 309 distributions required by the above features and set a cache for them.

For lisibility, we may gain to be able to use something like
```
          install-modules-args: --with-develop --feature=starman \
                                --feature=latex-pdf-ps --feature=openoffice \
                                --feature=xls --feature=edi
```

**Comment by shogo82148** [comment]
You can use the folding style of YAML: https://yaml.org/spec/1.2/spec.html#style/block/folded

```yaml
          install-modules-args: >
                    --with-develop --feature=starman
                    --feature=latex-pdf-ps --feature=openoffice
                    --feature=xls --feature=edi
```

**Comment by shogo82148** [comment]
released as v1.12.0

**Comment by lskatz** [comment]
Do you need `install-modules-args: --with-develop --with-configure` to have the cache?  Or will caching work without those options?
